server/utils/nunjucksSetup.ts

```diff
@@ -90,6 +90,13 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
 
   njkEnv.addFilter('pluralise', (word, number, appender) => (number === 1 ? word : `${word}${appender || 's'}`))
 
+  njkEnv.addFilter('pluraliseName', name => {
```

Comment:
Maybe pull this function out and a couple of tests. Also thanks from all us S ending names 👍